### ISSUE TITLE
UI problem: Save Path is very squashed (also: your FAQ page is broken)

### ISSUE THREAD

**Issue by LinuxOnTheDesktop** [flameshot-org/flameshot#1725]
The FAQ pag renders unreadably in both Firefox and Chrome, though it is hard to show why without a full-screen screenshot. Try visiting the site yourself.

**Flameshot version**
0.9.0
Compiled with QT 5.11.3

**Describe the bug**

![image](https://user-images.githubusercontent.com/47056543/126389126-5be790df-317d-441b-b73a-a1ec7c0ec34a.png)

**To Reproduce**
Open the relevant configuration pane.

**Expected behavior**
Obviously, not this.

**System Information**
OS: Linux Mint 20.2 x86_64 
Host: 2325L19 ThinkPad X230 
Kernel: 5.8.0-55-generic 
Resolution: 1366x768 @ 60.02Hz 
DE: Cinnamon 
WM: Mutter (Muffin) 
WM Theme: Mint-Y-Dark (Mint-Y-Dark) 
Theme: Mint-Y-Dark [GTK2/3] 
Icons: Paper [GTK2/3] 
CPU: Intel i5-3320M (4) @ 3.300GHz 
GPU: Intel 3rd Gen Core processor Graphics Controller 
Memory: 1953MiB / 7778MiB

**Comment by mmahmoudian** [comment]
@LinuxOnTheDesktop firefox has builtin screenshot capability and you can use that if needed.

All the symptoms you explained points to the direction that your computer has some fractional scaling issues. The giveaway was the Firefox as what we have used for our website (which is also opensource) is a very common statistic site generator without any complex css pr JavaScript, therefore, if your Firefox cannot handle that, I bet many other websites are also having the same problem.


Isn't this duplicate of #1697?

**Comment by LinuxOnTheDesktop** [comment]
It seems clear now that I should have opened two issues, one for the website rendering problem and one for the save path. I will close this issue and open two news ones. Admittedly I contributed to the confusion by the way that I ended up starting the bug report. Note though that as I said not only Firefox but also Chrome fails to render your FAQ page properly.

I have now opened the new issues. The one is #1726, the other #1727.
